desktop/apps/auction_support/client/registration_form.coffee

```diff
@@ -103,18 +110,19 @@ module.exports = class RegistrationForm extends ErrorHandlingForm
       else
         resolve()
 
+  # Lock the form- action.finally() callback executes when form submission is complete regardless of success
   loadingLock: ($element, action) ->
     return if $element.hasClass('is-loading')
     $element.addClass 'is-loading'
-    action().finally => $element.removeClass 'is-loading'
+    action().finally => $element.removeClass 'is-loading' unless @comboForm
```

Comment:
For situations like this its best to write a test that catches (or reproduces) the condition you're describing so things become more concrete and operable -- aside from that, try to break it manually? If successful, maybe rework or experiment a bit with a solution; if not, best to look for bigger 🐟  in the 🌊 🌊 🌊